classrooms/forms.py

```
@@ -0,0 +1,29 @@
+from django import forms
+from django.utils.text import slugify
+
+from .models import Classroom, ClassroomTeacher
```

Ruff (0.8.2)

`.models.ClassroomTeacher` imported but unused

Remove unused import: `.models.ClassroomTeacher`

(F401)